snapcraft/file_utils.py

```diff
@@ -335,6 +330,26 @@ def calculate_hash(path: str, *, algorithm: str) -> str:
     return hasher.hexdigest()
 
 
+def get_host_tool_path(*, command_name: str, package_name: str) -> pathlib.Path:
+    """Return the path command found on the host.
+
+    Using shutil.which() to find the host-provided command,
+    raises SnapcraftHostMissingToolError if not found.
+
+    :param command_name: command to find
+    :param package_name: package <command> is usually found in
```

Comment:
shouldn't `<command>` be just `command_name` in this docstring entry?

Reply:
done